thinc/backends/_custom_kernels.cu

```diff
@@ -161,6 +161,20 @@ __global__ void clipped_linear(T* Y, const T* X, double slope, double offset, do
 }
 
 
+template <typename T>
+__global__ void dish(T* Y, const T* X, int N)
+{
+    int _loop_start = blockIdx.x * blockDim.x + threadIdx.x;
+    int _loop_stride = blockDim.x * gridDim.x;
+
+    for (int i = _loop_start; i < N; i += _loop_stride)
+    {
+        T x = X[i];
+        Y[i] = 0.5 * x * (x / sqrt(1 + x * x) + 1);
```

Comment:
is this the super fast `sqrt` ?

Reply:
We don't really need anything like the good old fast inverse square root, since modern GPUs/CPUs have intrinsics for `sqrt` which makes computing the square root pretty fast. I think/guess nvcc would replace the square root by some inverse square root intrinsic (so the inner computation would change to something like `x * rsqrt(1 + x * x)`. For CPU, we can use their native reciprocal square root instructions once we support defining ops in terms of intrinsics.